src/videojs-http-streaming.js

```diff
@@ -243,13 +243,12 @@ const setupEmeOptions = ({
   audioMedia,
   mainPlaylists
 }) => {
-  if (!player.eme) {
-    return;
-  }
-
   const sourceOptions = emeKeySystems(sourceKeySystems, media, audioMedia);
 
-  if (!sourceOptions) {
+  if (!player.eme || !sourceOptions) {
+    if (sourceOptions) {
+      player.error({code: 5, message: 'DRM encrypted source cannot be decrypted without DRM plugin'});
```

Comment:
we probably don't want to change the message and just log this in the console. Since this message is user-facing, it'll mean we'll have to wordsmith it and then translate it.